encryption_works.md

```diff
@@ -120,21 +120,31 @@ When Snowden [was answering questions on Guardian's website](http://www.guardian
 
-OTR is used for two things: **encrypting the contents** of real-time instant message conversations and **verifying the identity** of people that you chat with. Identity verification is extremely important and something that many OTR users neglect to do. While OTR is much more user friendly that other forms of public key encryption, if you wish to use it securely you still need to understand how it works and what attacks against it are possible.
+### A Note About Gmail's "Hangout history" function.
 
-### Service Providers and Jabber
+When using Google Hangouts for instant messaging, there's a setting to disable "Hangout history," and it's useful to differentiate between this feature and what the software Off-the-Record offers.
 
-Using OTR only encrypts the contents of your chat conversations but not the metadata related to them. This metadata includes who you talk to and when and how often you talk to them. For this reason I recommend using a service that isn't known to collaborate with intelligence agencies. While this won't necessarily protect your metadata at least you have a chance of keeping it private.
+![Google's version of going off the record isn't secure.](http://i.imgur.com/LkXGr6s.png)
 
-I also recommend you use an XMPP (also known as Jabber) service. Like email, Jabber is a federated, open protocol. Users of [riseup.net](https://www.riseup.net/en/chat)'s Jabber service can chat with users of [jabber.ccc.de](https://web.jabber.ccc.de/)'s service as well as [jabber.org](http://www.jabber.org/)'s service.
+Essentially, disabling "Hangout history" function does not display a conversation log to the user. This takes the conversation off the record in the literal sense, but users aren't given any reason to believe Google isn't still saving or reading the chat logs. In comparison, conversations over Google Talk or Jabber which have the Off the Record software layer installed aren't readable to anyone except the participants, including Google. This is far more secure.
 
-### OTR Clients
+If you haven't yet updated from Google Talk to Hangouts, the feature is confusingly also called "off the record." 
 
-To use OTR you'll need to download software. If you use Windows you can download and install [Pidgin](https://pidgin.im/) and separately the [OTR plugin](http://www.cypherpunks.ca/otr/). If you use GNU/Linux you can install the pidgin and pidgin-otr packages. You can read through [documentation on how to set up your Pidgin accounts with OTR](http://www.cypherpunks.ca/otr/index.php#docs). If you use Mac OS X you can download and install [Adium](http://adium.im/), which is a free software chat client that includes OTR support. You can read the [official documentation](http://adium.im/help/pgs/AdvancedFeatures-OTREncryption.html) on how to get set up with OTR encryption with Adium.
```

Comment:
I say we keep all the Gmail stuff you put in, but also what Micah has about OTR clients and Jabber servers (maybe stealing some of his language from his recent Intercept piece and then linking to the rest). For the reasons stated above, it's more important to include jabber than it is gchat for metadata and anonymity purposes purposes. 

Basically, this section looks good, it's just a bit out of order.